header.php

```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
 <html <?php language_attributes(); ?>>
-<head>
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width">
-	<title>Setupshop0.1</title>
+  <head>
+	  <meta charset="utf-8">
+	  <meta name="viewport" content="width=device-width">
+    <img src="C:\Bitnami\wampstack-5.6.25-0\apache2\htdocs\Setupshop\wp-content\uploads\2016\09">
```

Comment:
Noted! ...this was for testing the image background. I'll give it a go just to see, otherwise, I've decided against a background cover image.